packages/flame/lib/src/components/core/component.dart

```diff
@@ -989,7 +988,7 @@ class Component {
   /// A key that can be used to identify this component in the tree.
   ///
   /// It can be used to retrieve this component from anywhere in the tree.
-  final ComponentKey? _key;
+  final ComponentKey? key;
```

Comment:
I wonder if we should call it componentKey for the same reason you have exposed; it is possible other people are using key for something else on their components, and this would clash. no strong opinion though

Reply:
I think it is rare enough, I've never seen it anywhere else except in the very new flame_riverpod instance